async-http-client-backend/monix/src/test/scala/sttp/client/asynchttpclient/monix/AsyncHttpClientMonixHttpTest.scala

```diff
@@ -20,4 +20,6 @@ class AsyncHttpClientMonixHttpTest extends HttpTest[Task] with CancelTest[Task,
       .onErrorRecover {
         case _: TimeoutException => None
       }
+
+  override def throwsExceptionOnUnsupportedEncoding = false
```

Comment:
shouldn't this be added to all async-http-client tests?

Reply:
It should, fixed :)